client/configuration_variable.go

```diff
@@ -41,7 +41,7 @@ func (c *ConfigurationVariableSchema) ResourceDataSliceStructValueWrite(values m
 
 type ConfigurationVariable struct {
 	ScopeId        string                       `json:"scopeId,omitempty"`
-	Value          string                       `json:"value"`
+	Value          string                       `json:"value" tfschema:"-"`
```

Comment:
this avoid writing the value to the schema. Special handling is required instead.